src/main/java/qna/domain/Answers.java

```diff
@@ -0,0 +1,69 @@
+package qna.domain;
+
+import org.hibernate.annotations.Where;
+import qna.CannotDeleteException;
+
+import javax.persistence.Embeddable;
+import javax.persistence.FetchType;
+import javax.persistence.OneToMany;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+@Embeddable
+public class Answers {
+
+    @OneToMany(fetch = FetchType.LAZY, mappedBy = "question")
+    @Where(clause = "deleted = false")
+    private List<Answer> answers = new ArrayList<>();
+
+    protected Answers() {
+    }
+
+    public Answers(List<Answer> answers) {
+        this.answers = answers;
+    }
+
+    public static Object of(Answer... answers) {
```

Comment:
`Object` 보다는 정적 팩토리 `method` 이므로 `Answers`가 더 명확할 것 같습니다 😄 
```suggestion
    public static Answers of(Answer... answers) {
```